panda.py

```diff
@@ -0,0 +1,224 @@
+from pxr import Usd, Vt
```

Comment:
Add a header and be proud of your work! :wink: 
```python
# 
#  USD -> ARaaS Rigging Export Tool
#  Author: Harsh Dhruva, 2021 
#
```

Reply:
Also, this file is indeed specific to the panda for now, but as the program changes (does not need to be for this PR!) you may wish to change it from `panda.py` to `usd_to_araas.py`